src/titiler/application/titiler/application/main.py

```diff
@@ -36,17 +56,27 @@
 )
 
 if not api_settings.disable_cog:
-    app.include_router(cog.router, prefix="/cog", tags=["Cloud Optimized GeoTIFF"])
+    app.include_router(
+        cog.router,
+        prefix=api_settings.path_prefix + "/cog",
```

Comment:
So these `prefix=api_settings.path_prefix + ...` changes are the meat of this PR.